roles/nvidia-network-operator/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+---
+# Deploy network operator related tasks on master node
+#
+
+- name: label the nodes
+  shell: kubectl label --overwrite nodes {{ item }} node-role.kubernetes.io/worker=
+  with_items: "{{ groups['kube-node']}}"
+
+- name: Install openshift
```

Comment:
Rather than install this here, add the existing openshift role as a dep. See roles/nvidia-gpu-operator/meta/main.yml for an example.

Reply:
I actually tried that existing role but failed, we can take a closer look if taking that approach.